commit 70a3b9ac0fffc05cbe541164c097f51040addc8c (refs/changes/39/239/2)
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2016-08-05T22:37:38+00:00 (3 years, 2 months ago)
    
    OS-5580 disable docs in kvm-cmd
    Reviewed by: Cody Mello <melloc@joyent.com>
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/build.sh b/build.sh
index 7c3907b..318f51c 100755
--- a/build.sh
+++ b/build.sh
@@ -51,6 +51,7 @@ XLDFLAGS="${XLDFLAGS} -lz -lm -lc -lvnd"
     --disable-brlapi \
     --disable-curl \
     --enable-debug \
+    --disable-docs \
     --enable-kvm \
     --enable-kvm-pit \
     --enable-vnc-png \
